src/main/java/it/rate/webapp/services/InterestService.java

```diff
@@ -27,12 +30,15 @@ public Optional<Interest> findInterestById(Long id) {
     return interestRepository.findById(id);
   }
 
+  public Interest getById(Long id) {

```

Comment:
this is related to my comment above, if you end up changing it back, remove this as well.